lib/src/highlighter.dart

```diff
@@ -176,6 +185,14 @@ class HighlighterTheme {
 
   HighlighterTheme._({required TextStyle wrapper}) : _wrapper = wrapper;
 
+  /// load a [HighlighterTheme] from a [json] string.
```

Comment:
Just a tiny fix to the formatting of the comment.

```dart
/// Load a [HighlighterTheme] from a JSON string.
```

Missing captial L and json should be in brackets as it is not referring to a Dart entity.

Reply:
sorry for missing this changes 🙏